process/transaction/interceptedTransaction.go

```diff
@@ -137,6 +148,55 @@ func (inTx *InterceptedTransaction) CheckValidity() error {
 	return nil
 }
 
+func (inTx *InterceptedTransaction) verifyIfRelayedTx(tx *transaction.Transaction) error {
```

Comment:
Tests missing for this function

Reply:
agreed after first reviews will do it. may merge after disabling this txtype